### PR TITLE
Fix warnings in tests

### PR DESCRIPTION
**Pull request by lialila**

### CODE REVIEW

File: src/slices/ProjectsMap/mapboxStyle.ts [hunk 1]
```diff
@@ -142,7 +142,7 @@ const mapboxStyle = css`
   .mapboxgl-ctrl button:disabled .mapboxgl-ctrl-icon {
     opacity: 0.25;
   }
-  .mapboxgl-ctrl-group button:first-child {
+  .mapboxgl-ctrl-group button:first-of-type {
```
Comment (lukasbals) [inline src/slices/ProjectsMap/mapboxStyle.ts:145]: Are you sure that this is the same behaviour as before? 

From my understanding, it's now the first button and before it was the first child of the button. But maybe I'm wrong.

File: src/slices/TextCarousel/TextCarousel.tsx [hunk 2]
```diff
@@ -51,10 +51,10 @@ export const TextCarousel: React.FC<TextCarouselProps> = ({
         <CarouselInnerContainer
           drag="x"
           dragConstraints={containerRef}
-          numberOfItems={slice.slides.length}
+          numberofitems={slice.slides.length}
```
Comment (lukasbals) [inline src/slices/TextCarousel/TextCarousel.tsx:54]: Is `numberOfItems` a reserved keyword? Or why does it cause a problem? 
Or is `numberofitem` a standard that should be used? 

If `numberOfItems` is a reserved keyword I would go with `numItems`.
If `numberofitems` is a standard I would go with it.

File: package-lock.json
Comment (lukasbals) [inline package-lock.json:1]: Are those changes needed? 

What Node version do you use on your local machine? These seem to be changes that arise if you have a different Node version installed.

Comment (lialila) [reply package-lock.json:1]: I use the 20. Node.js, changed it to 18! that's right?

File: src/slices/FullWidthImageSlider/FullWidthImageSlider.tsx [hunk 3]
```diff
@@ -97,7 +97,7 @@ export const FullWidthImageSlider: React.FC<FullWidthImageSliderProps> = ({
         </SliderInnerContainer>
       </SliderContainer>
 
-      <ButtonsContainer show={allowScroll && !isOpen}>
+      <ButtonsContainer show={allowScroll && !isOpen ? 'true' : 'false'}>
```
Comment (lukasbals) [inline src/slices/FullWidthImageSlider/FullWidthImageSlider.tsx:100]:
```suggestion
      <ButtonsContainer show={(allowScroll && !isOpen).toString()}>
```

File: src/slices/ProjectsMap/mapboxStyle.ts (hunk 1 above)
Comment (lialila) [reply src/slices/ProjectsMap/mapboxStyle.ts:145]: Yes, it should work all right:
https://css-tricks.com/almanac/selectors/f/first-child/#:~:text=Using%20%3Afirst%2Dchild,in%20the%20HTML.

File: src/slices/TextCarousel/TextCarousel.tsx (hunk 2 above)
Comment (lukasbals) [reply src/slices/TextCarousel/TextCarousel.tsx:54]: Ok. Doesn't make sense to me. But ok.

File: package-lock.json
Comment (lukasbals) [reply package-lock.json:1]: Yes, I think we all use v18. But I there are still the changes with the "peer". 

Can you please try to undo the changes in the `package-lock.json` file and run `npm ci` with Node.js version 18 and check if there are changes in the `package-lock.json` file? I think there should be no changes 🤔

Comment (lialila) [reply package-lock.json:1]: yes! thanks

**Comment by lukasbals** [comment]
You also need to update the commit message to `chore: Fix warnings in tests`. The prefixes in the commit messages are there for semantic release to know what to do. `chore: ` means that it only does a release when something changes in the package. Other prefixes are `fix: ` for a bug fix release or `feat: ` to trigger a new feature release.

**Pull request merged**